### ISSUE TITLE
🪝 Fish Hook icon PLZ :)

### ISSUE THREAD

**Issue by jonahgoldsaito** [phosphor-icons/homepage#504]
**Describe the requested icon(s)**

A fish hook! 
There are anchors, and fish, but no hook.

Of course there's the literal usage, but more importantly there are many instances where hooking something, being hooked, are useful.

I'm a designer at Khan Academy (WE LOVE PHOSPHOR and see you listed us on your landing page!!!), and "Lesson Hooks" are something that are part of teachers' lives, and their professional, day-to-day vocabulary in lesson planning. How do we hook a lesson to a topic or theme that resonates with the students and makes it relevant. 

**References**

🪝 🪝 🪝 🪝 🪝 🪝 🪝 🪝 🪝 🪝 🪝  😅
Many photos on wikipedia [here](https://en.wikipedia.org/wiki/Fish_hook).

Screenshot of the icon I tried my hand out (in case that's useful, attaching SVGs below)

![hooksScreenshot](https://github.com/phosphor-icons/homepage/assets/1239984/7215c41e-f649-4ff2-aacd-3085a8a8ee32)

**Additional context**
 
Our design team and engineers are tied so heavily to your library that we very rarely introduce custom icons so that our whole build system can work smoothly. It's all about the pipeline! :) So it'd be awesome to have this or something like it in your library! LOVE AND APPRECIATE YOUR WORK

![Fishing Hook Thin](https://github.com/phosphor-icons/homepage/assets/1239984/de0695a1-64c4-49ca-b354-bc6a5bd2630e)
![Fishing Hook Light](https://github.com/phosphor-icons/homepage/assets/1239984/086167f3-ba40-4cbf-899b-05311792f95a)
![Fishing Hook Regular](https://github.com/phosphor-icons/homepage/assets/1239984/f753317d-8639-4e50-9a15-71788559bfce)
![Fishing Hook Bold](https://github.com/phosphor-icons/homepage/assets/1239984/9fab539f-9aa1-4788-840f-62459169a468)
![Fishing Hook Fill](https://github.com/phosphor-icons/homepage/assets/1239984/93f3385a-c0a0-4d2e-b969-7fb6158f7f84)
![Fishing Hook Duotone](https://github.com/phosphor-icons/homepage/assets/1239984/4d78434e-2e6e-4ede-92da-c45fb9f6b8c6)

❤️ - an admirer

**Comment by ryanshumway** [comment]
Yes! I was looking for a fishing hook and there isn't one. There's Anchor, and fish.. specifically in my use case we need to represent "Phishing" (and a hook would be perfect).

**Comment by minoraxis** [comment]
Prioritizing for our next release :)
